src/module/item/SR5ItemSheet.ts

```diff
@@ -189,44 +189,33 @@ export class SR5ItemSheet extends foundry.appv1.sheets.ItemSheet {
         /**
          * Reduce nested items into typed lists.
```

Comment:
That comment doesn´t match the new algorithm anymore:

Groups items into a list by type: grouped.weapon = [...], grouped.spell = [...]